docs/user-guide/manual-install.md

```diff
@@ -0,0 +1,210 @@
+---
+    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+---
+# Quick Start Installation Guide
```

Comment:
Why is this called the Quick Start if it's the manual process?

Reply:
fixed